src/main/kotlin/http/KtorServer.kt

```diff
@@ -94,6 +92,14 @@ private suspend fun sendNotFoundResponse(call: ApplicationCall) {
     )
 }
 
+private suspend fun sendNotAuthorized(call: ApplicationCall, realm: String) {
```

Comment:
I have started working on this one. It's a bit tricky, because in the current scope it is not possible to tell if basic auth is enabled or not. Trying to figure out the least invasive wat of solving these with regard to current structore of the code.

Reply:
This is a separate pull request now https://github.com/zensum/leia/pull/18